extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -18,9 +18,13 @@ export const NetCoreInstallLocationKey: string = 'netCoreSDKLocation';
 export const NetCoreDoNotAskAgainKey: string = 'netCoreDoNotAsk';
 export const NextCoreNonWindowsDefaultPath = '/usr/local/share';
 export const NetCoreInstallationConfirmation: string = localize('sqlDatabaseProjects.NetCoreInstallationConfirmation', "The .NET Core SDK cannot be located. Project build will not work. Please install .NET Core SDK version 3.1 or update the .Net Core SDK location in settings if already installed.");
+export const NetCoreSupportedVersionInstallationConfirmation: string = localize('sqlDatabaseProjects.NetCoreSupportedVersionInstallationConfirmation', "The .NET Core SDK that was found isn't a supported version. Project build will not work. Please install .NET Core SDK version 3.1 or update the .Net Core SDK supported version location in settings if already installed.");
```

Comment:
can the currently installed .Net Core SDK version be added to this message?

Reply:
Added it in the message.